html/css/main.css

```diff
@@ -406,4 +426,73 @@ header li {
     font-style: normal;
     border-radius: 4px;
     font-size:0.8em;
-}
\ No newline at end of file
+}
+
+/* code highlighting */
+main {min-width:0;} /* trick to avoid code blocks expanding main area */
+pre > code.sourceCode { white-space: pre; position: relative; }
+pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
+pre > code.sourceCode > span:empty { height: 1.2em; }
+.sourceCode { overflow: visible; background: var(--bkg-color-pre);}
+code.sourceCode > span { color: var(--code-color); text-decoration: inherit; }
+div.sourceCode { margin: 1em 0; }
+pre.sourceCode { margin: 0; }
+@media screen {
+div.sourceCode { overflow: auto;}
+}
+@media print {
+pre > code.sourceCode { white-space: pre-wrap; }
+pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
+}
+pre.numberSource code
+  { counter-reset: source-line 0; }
+pre.numberSource code > span
+  { position: relative; left: -4em; counter-increment: source-line; }
+pre.numberSource code > span > a:first-child::before
+  { content: counter(source-line);
+    position: relative; left: -1em; text-align: right; vertical-align: baseline;
+    border: none; display: inline-block;
+    -webkit-touch-callout: none; -webkit-user-select: none;
+    -khtml-user-select: none; -moz-user-select: none;
+    -ms-user-select: none; user-select: none;
+    padding: 0 4px; width: 4em;
+    olor: #aaaaaa;
```

Comment:
is this line correct?

Reply:
It was a mistake, though we don't use line numbers so I didn't notice.